rollup.config.js

```diff
@@ -13,7 +13,7 @@ export default [
       {
         file: 'dist/bundles/umd/base.js',
         format: 'umd',
-        name: 'CloudinaryBaseSDK',
+        name: 'CLD_URL_GEN',
```

Comment:
I would change this to lower case. `cld_url_gen`

Reply:
CldUrlGen?

Reply:
Changed to `CldUrlGen`